### PR TITLE
[MIG] dead_mans_switch_client: Migration to 10.0

### PR DESCRIPTION
**Pull request by mlaitinen**
Note that I didn't change the Runbot URL. What should I put there?

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Hi @mlaitinen 

You should put `https://runbot.odoo-community.org/runbot/149/10.0`. 149 is the id of server-tools repository and 10.0 is the odoo version.

**Comment by mlaitinen** [comment]
@lreficent I tried that but the URL isn't working. I get:

> This page isn’t working
> 
> 3310280-10-0-a5cc2c.runbot2.odoo-community.org redirected you too many times.

**Comment by LoisRForgeFlow** [comment]
@mlaitinen It is working now!

**Comment by StefanRijnhart** [comment]
Now it's not working, but that is not your problem. It is still the right URL.

File: dead_mans_switch_client/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 # © 2015-2016 Therp BV <http://therp.nl>
+# © 2017 Avoin.Systems - Miku Laitinen
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Dead man's switch (client)",
-    "version": "9.0.1.0.1",
-    "author": "Therp BV,Odoo Community Association (OCA)",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV, Avoin.Systems, Odoo Community Association (OCA)",
```
Comment (StefanRijnhart) [inline dead_mans_switch_client/__manifest__.py]: I do appreciate the port to version 10, and I think adding yourself as a contributor is fine, but personally I would not put myself in the author's list for a change like this.

Comment (mlaitinen) [reply dead_mans_switch_client/__manifest__.py]: I agree with you. I don't contribute that often to OCA so I'm not always up to the guidelines.

I will remove our company from the authors. Is it ok if I keep the mention in other files?

**Comment by mlaitinen** [comment]
I fixed the Runbot URL and removed us from the author list.

**Review by StefanRijnhart — APPROVED** [review]

**Review by hbrunn — APPROVED** [review]

**Pull request merged**